src/chrome/content/rules/EDP.pt.xml

```diff
@@ -40,6 +41,8 @@ www.cdme.edp.pt
 	<target host="sinergie.misy.eu" />
 	<target host="sts.edp.pt" />
 	<target host="ease.edp.pt" />
+	<target host="www.edp.pt" />
 	
-	<rule from="^http:" to="https:" />
+	<rule from="^" to="https://www" />
+	<rule from="www" to="https:www" />
```

Comment:
This won't work.
Use `<rule from="^http://edp\.pt/" to="https://www.edp.pt/" />` and `<rule from="^http:" to="https:" />`.